deploy/registration-service.yaml

```diff
@@ -98,6 +98,7 @@ objects:
               ports:
                 - containerPort: 8080
                 - containerPort: 8081
+                - containerPort: 8082
```

Comment:
This file is used only in dev environments. The one which is used in production (and e2e tests) are defined here: https://github.com/codeready-toolchain/host-operator/blob/master/deploy/registration-service/registration-service.yaml
Please create a PR to update that file too.

But what I'm wonder is why the e2e-test still pass? Do we have any e2e tests for this metric service? Without updating https://github.com/codeready-toolchain/host-operator/blob/master/deploy/registration-service/registration-service.yaml the e2e-tests for this PR should fail...

Reply:
could we also add names to the container ports (see for example https://kubernetes.io/docs/concepts/services-networking/service/#field-spec-ports) to make them more explicit, and reuse these names in the services as well?

Reply:
> This file is used only in dev environments.

It's actually not used even in dev environment. This file is a legacy thing from the time when we were able to deploy reg-service on its own - we don't do that anymore (or at least I'm not aware of anyone who would be doing that)

Reply:
but anyway, Alexey is right that you need to create PR in host-repo with this change. Let's do it in two steps:
1. create PR in host-operator repo changing only the values in the template (I don't think that we verify the actual content of the deployment in e2e tests)
3. when the host-operator PR is merged, create a PR in e2e tests that will be paired with this reg-service one that will verify that the `Service` is created.

Reply:
Sorry for the late reply,
@MatousJobanek @alexeykazakov Thanks for your comments and catching the missing host-repo changes. I did have the changes in host-repo, that's how i was able to test it locally indeed, but had missed creating the PR on host-repo. I've updated the host-operator and e2e PR in the description.